src/Microsoft.AspNetCore.WebHooks.Receivers/Metadata/IWebHookBodyTypeMetadataService.cs

```diff
@@ -1,14 +1,27 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 namespace Microsoft.AspNetCore.WebHooks.Metadata
 {
     /// <summary>
-    /// Metadata describing the request body type an action expects. <see cref="IWebHookMetadata"/> service
-    /// implementations that contain this information must implement this interface and not just
-    /// <see cref="IWebHookBodyTypeMetadata"/>.
+    /// Metadata describing the request body type a receiver requires. Implemented in a <see cref="IWebHookMetadata"/>
+    /// service for all receivers.
     /// </summary>
-    public interface IWebHookBodyTypeMetadataService : IWebHookBodyTypeMetadata, IWebHookReceiver
+    /// <remarks>
+    /// <see cref="ApplicationModels.WebHookModelBindingProvider"/> sets <see cref="Mvc.ModelBinding.BindingInfo"/>
+    /// properties based on this metadata or <see cref="IWebHookBodyTypeMetadata"/> and
+    /// <see cref="IWebHookBindingMetadata"/>. <see cref="Filters.WebHookVerifyBodyTypeFilter"/> confirms the request
+    /// body type based on this metadata. <see cref="Filters.WebHookEventMapperFilter"/> uses this metadata to decide
+    /// how to parse the request body and how to interpret
+    /// <see cref="IWebHookEventFromBodyMetadata.BodyPropertyPath"/>.
+    /// </remarks>
+    public interface IWebHookBodyTypeMetadataService : IWebHookMetadata, IWebHookReceiver
```

Comment:
I think I'm really confused about something and we should just discuss, what's the difference between this and `IWebHookBodyTypeMetadata`?